site-content/source/modules/ROOT/pages/native_protocol.adoc

```diff
@@ -3,21 +3,81 @@
 
 == Native Protocol Version 3
 
-[source, plaintext]
-----
-include::example$TEXT/native_protocol_v3.spec[Version 3]
-----
+[source, js]
+++++
+<div id="contentv3" class="doc-container"></div>
+<iframe id="iframev3" src="native_protocol_v3.html" style="display:none;"></iframe>
+++++
 
 == Native Protocol Version 4
 
-[source, plaintext]
-----
-include::example$TEXT/native_protocol_v4.spec[Version 4]
-----
+[source, js]
+++++
+<div id="contentv4" class="doc-container"></div>
+<iframe id="iframev4" src="native_protocol_v4.html" style="display:none;"></iframe>
+++++
 
 == Native Protocol Version 5
 
-[source, plaintext]
-----
-include::example$TEXT/native_protocol_v5.spec[Version 5]
-----
+[source, js]
+++++
+<div id="contentv5" class="doc-container"></div>
+<iframe id="iframev5" src="native_protocol_v5.html" style="display:none;"></iframe>
+++++
+
+[source, js]
+++++
+<script>
```

Comment:
Since we moved the cqlprotodoc code here, I wonder if including the content using Javascript is still needed or whether we should modify the `template.gohtml` / cqlprotodoc output and include the html files directly.

Reply:
I think it's better to keep things as they are because the script placed in the .adoc template appears more readable to me. Also, we avoid changing the website template by keeping native_protocol.html as the location where the generated pages are merged.